recipes/boost/all/conandata.yml

```diff
@@ -2,41 +2,48 @@ sources:
   1.69.0:
     url: [
       "https://dl.bintray.com/boostorg/release/1.69.0/source/boost_1_69_0.tar.bz2",
```

Comment:
Remove Bintray already? I don't see the point of leaving it there for another 2 weeks just to remove it then

Reply:
I think more mirrors is harmless, but increases reliability, in case of boostorg.jfrog.io is unreachable for whatever reason, dl.bintray.com could be used as a fallback